python/pyarrow/_csv.pyx

```diff
@@ -1337,6 +1355,17 @@ cdef class WriteOptions(_Weakrefable):
     def delimiter(self, value):
         deref(self.options).delimiter = _single_char(value)
 
+    @property
+    def quoting_style(self):
+        """
+        Quoting style for CSV writing.
+        """
+        return deref(self.options).quoting_style
```

Comment:
Does this actually return the same values as expected by the constructor and property setter?

Reply:
It converted to `int` since that's what the C++ enums default to I think. Fixed by doing the reverse of `unwrap_quoting_style`.